cmake/Dependencies.cmake

```diff
@@ -539,6 +539,12 @@ if(USE_XNNPACK AND NOT USE_SYSTEM_XNNPACK)
     set(__caffe2_CMAKE_POSITION_INDEPENDENT_CODE_FLAG ${CMAKE_POSITION_INDEPENDENT_CODE})
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+    if(Win32)
+      # Disable libm dependency explicitly to avoid symbol conflict for XNNPACK as
+      # Windows runtime has provided the math functions - #134989
+      set(XNNPACK_BUILD_WITH_LIBM OFF CACHE BOOL "")
```

Comment:
Hi @xuhancn from my understanding we are trying to fix issue with Intel GPU: https://github.com/pytorch/pytorch/issues/134989 but looks like this this change is more general, it will affect all windows builds. Can we restrict it only to XPU ?

Reply:
Hi @atalman I think it is not only for Intel XPU. Windows is not as same as Linux, Windows is designed implement math functions in its runtime lib, but not libm. So any situlation, which contains `libm`, that will occur this issue on Windows.

Reply:
What are performance considerations for this ?